test/unit/search/helper.hpp

```diff
@@ -43,14 +42,16 @@
 #include <algorithm>
 #include <vector>
 
+#include <seqan3/alphabet/all.hpp>
+
 namespace seqan3
 {
 
 template <typename T>
 std::vector<T> sort(std::vector<T> v)
 {
     std::sort(v.begin(), v.end());
-    v.erase(std::unique(v.begin(), v.end()), v.end()); // TODO: remove once duplicates are filtered
+    v.erase(std::unique(v.begin(), v.end()), v.end());
     return v;
 }
```

Comment:
what does this do? :thinking: 

Remove all elements that don't appear multiple times?